docs/MASTG.md

```diff
@@ -15,7 +15,7 @@ The **OWASP Mobile Application Security Testing Guide (MASTG)** is a comprehensi
 
 <br>
 
-[:material-download: Download the MASTG](https://github.com/OWASP/owasp-mastg/releases/latest/download/OWASP_MASTG-v1.5.0.pdf){ .md-button .md-button--primary }
+[:material-download: Download the MASTG](https://github.com/OWASP/owasp-mastg/releases/download/v1.6.0/OWASP_MASTG.pdf){ .md-button .md-button--primary }
```

Comment:
Thanks a lot! I'm changing it like this so that we don't have to update the version anymore.

```suggestion
[:material-download: Download the MASTG](https://github.com/OWASP/owasp-mastg/releases/latest/download/OWASP_MASTG.pdf){ .md-button .md-button--primary }
```